wagtailvideos/templates/wagtailvideos/chooser/results.html

```diff
@@ -16,7 +16,7 @@ <h2>{% trans "Latest videos" %}</h2>
     <ul class="listing horiz images chooser">
         {% for video in videos %}
             <li>
-                <a class="image-choice" href="{% if will_select_format %}{% url 'wagtailvideos:chooser_select_format' video.id %}{% else %}{% url 'wagtailvideos:video_chosen' video.id %}{% endif %}">
+                <a class="video-choice" href="{% if will_select_format %}{% url 'wagtailvideos:chooser_select_format' video.id %}{% else %}{% url 'wagtailvideos:video_chosen' video.id %}{% endif %}">
```

Comment:
the `image-choice` class was used so the styles from the default wagtail css is applied to the chooser without having to reimplement it. Doesn't look like any styling was added in the pr so this will need to be reverted.

https://github.com/wagtail/wagtail/blob/c447f75bd4a74566db71a45037067692b4342b0d/wagtail/admin/static_src/wagtailadmin/scss/components/_listing.scss#L424

Reply:
I see this is used for the modal set up. You can add a data attribute instead for the selectors so the styles are still applied